lib/preload.js

```diff
@@ -437,16 +434,19 @@ class Preloader extends EventEmitter {
       // Pull the image
       return innerDockerProgress.pull(appData.imageId, (event) => {
         // Emit progress events while pulling
-        percentage = event.percentage
         this._progress(pullingProgressName, event.percentage)
       })
     })
     .then(() => {
-      if (percentage !== 100) {
-        return
-      }
```

Comment:
Ok, I don't understand a lot of the context of this. Presumably this code used to not signal 'done' if percentage !== 100 for some reason before though. Why was that, and why is it no longer relevant?

Reply:
@pimterry it's dead code. It was introduced by https://github.com/resin-io/resin-preload/commit/75addc2a291674d92250da495cc56f3db167026a .
This never happens.
Even if it happened, we should still signal the python process that we're done pulling.